### DISCUSSION TITLE
复制单元格内容到github评论框, 出现<meta charset="utf-8"> 标签， 怎么移除呢，或者有哪个参数可以避免呢

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#2214] [Q&A] [answered]
Cltr+c 、Ctrl+v ,  @antv/s2 version: 1.30.0

![image](https://github.com/antvis/S2/assets/15354455/c2fa20aa-dbca-4126-bbdd-915d1858a17b)

![image](https://github.com/antvis/S2/assets/15354455/ec347c64-a639-41b9-a07a-c1da5a3f3afb)

**Comment by Chasen-Zhang** [comment]
![image](https://github.com/antvis/S2/assets/15354455/bb71f350-9f43-4f93-bb58-c4ed4df298f2)
试了下官方文档的示例，也有这个问题。

https://s2.antv.antgroup.com/zh/examples/interaction/basic/#hover

**Comment by lijinke666** [comment, marked as answer]
有 meta 是因为复制出来是 HTML 格式, 这样粘贴其他富文本编辑器中可以显示出表格, 符合预期, 问题是啥? 带来了什么问题?

S2 会在复制的时候往剪贴板写入两种类型的元数据

- `text/html`
- `text/plain`

粘贴的时候, 取决于接收方选择什么类型的数据, 这里 (GitHub) 富文本接受的 `text/html`, 对于 Excel 之类的就是 `text/plain`, 即带制表符 `\t` 的纯文本

所以不需要移除, 也不存在什么参数避免, 更多背景请查看 https://juejin.cn/post/7160976446498799652

![image](https://github.com/antvis/S2/assets/21015895/ad217581-d2d7-45e1-91e4-4850f76d2a1d)
![image](https://github.com/antvis/S2/assets/21015895/d3e9cd6b-08d9-456a-9bf1-6b5779512b1b)

**Reply by Chasen-Zhang** [reply]
可以复制纯文本不，不是HTML 格式呢 @lijinke666

**Reply by lijinke666** [reply]
为什么要控制粘贴到 GitHub 的行为, 不是很明白, HTML 格式不是更好吗?

**Reply by lijinke666** [reply]
应该回复的很清楚了吧, 是复制的时候就带了 文本和 HTML 两种格式了, 怎么用取决于使用方怎么使用, 不是我们控制的

![image](https://github.com/antvis/S2/assets/21015895/68beab44-9c24-4581-a92e-ca39bc896885)

**Reply by Chasen-Zhang** [reply]
好的，感谢
